action.yml

```diff
@@ -44,6 +47,7 @@ runs:
     - --parse-mode
     - ${{ inputs.parse-mode }}
     - --to=${{ inputs.to }}
+    - --thread_id=${{ inputs.thread_id }}
```

Comment:
Let's call it `thread-id`, this way it's aligned to the way the other flags are written.